updater/lib/dependabot/updater.rb

```diff
@@ -733,9 +733,10 @@ def generate_dependency_files_for(updated_dependencies)
         logger_info("Updating #{dependency_names.join(', ')}")
       end
 
-      # Removal is only supported for transitive dependencies which are removed as a
-      # side effect of the parent update
-      deps_to_update = updated_dependencies.reject(&:removed?)
+      # Ignore dependencies that are tagged as information_only. These will be
+      # updated indirectly as a result of a parent dependency update and are
+      # only included here to be included in the PR info.
+      deps_to_update = updated_dependencies.reject { |d| d.metadata[:information_only] }
```

Comment:
I had the thought of whether it makes sense to move this to a dependency method like `#informational_only?` or something, but I'm not sure if other ecosystems will need something like this so probably premature. I just thought that `deps_to_update = updated_dependencies.reject(&:informational_only?)` would read a bit better.

Reply:
Thanks! I was on the fence about adding that but you've persuaded me!